player.py

```diff
@@ -3,20 +3,11 @@ def __init__(self):
         self.hand = []
 
     def calculate_score(self):
-        num_of_aces = 0
-        calc_score = 0
+        self.hand.sort(key=lambda x: x.score)
+        hand_score = 0
```

Comment:
Everything in here is much cleaner. It's still doing more than we need it to do. But we'll get there. Like I've said, our goal for this method is to be one line of code. This is closer to that than we were.